src/vaadin-split-layout.js

```diff
@@ -0,0 +1,360 @@
+ *
+ * ### Split Layout Element Height
+ *
+ * `<vaadin-split-layout>` element itself is a flex container. It does not inherit
+ * the parent height by default, but rather sets its height depending on the
+ * content.
+ *
+ * You can use CSS to set the fixed height for the split layout, as usual with any
+ * block element:
+ *
+ * ```html
+ * <vaadin-split-layout style="height: 200px;">
+ *   <div>First content element</div>
+ *   <div>Second content element</div>
+ * </vaadin-split-layout>
+ * ```
+ *
+ * It is possible to define percentage height as well. Note that you have to set
+ * the parent height in order to make percentages work correctly. In the following
+ * example, the `<body>` is resized to fill the entire viewport, and the
+ * `<vaadin-split-layout>` element is set to take 100% height of the `<body>`:
+ *
+ * ```html
+ * <body style="height: 100vh; margin: 0;">
+ *   <vaadin-split-layout style="height: 100%;">
+ *     <div>First</div>
+ *     <div>Second</div>
+ *   </vaadin-split-layout>
+ * </body>
+ * ```
+ *
+ * Alternatively, you can use a flexbox layout to make `<vaadin-split-layout>`
+ * fill up the parent:
+ *
+ * ```html
+ * <body style="height: 100vh; margin: 0; display: flex;">
+ *   <vaadin-split-layout style="flex: 1;">
+ *     <div>First</div>
+ *     <div>Second</div>
+ *   </vaadin-split-layout>
+ * </body>
+ * ```
+ *
+ * ### Initial Splitter Position
+ *
+ * The initial splitter position is determined from the sizes of the content elements
+ * inside the split layout. Therefore, changing `width` on the content elements
+ * affects the initial splitter position for the horizontal layouts, while `height`
+ * affects the vertical ones.
+ *
+ * Note that when the total size of the content elements does not fit the layout,
+ * the content elements are scaled proportionally.
+ *
+ * When setting initial sizes with relative units, such as percentages, it is
+ * recommended to assign the size for both content elements:
+ *
+ * ```html
+ * <vaadin-split-layout>
+ *   <div style="width: 75%;">Three fourths</div>
+ *   <div style="width: 25%;">One fourth</div>
+ * </vaadin-split-layout>
+ * ```
+ *
+ * ### Size Limits
+ *
+ * The `min-width`/`min-height`, and `max-width`/`max-height` CSS size values
+ * for the content elements are respected and used to limit the splitter position
+ * when it is dragged.
+ *
+ * It is preferred to set the limits only for a single content element, in order
+ * to avoid size conflicts:
+ *
+ * ```html
+ * <vaadin-split-layout>
+ *   <div style="min-width: 50px; max-width: 150px;">First</div>
+ *   <div>Second</div>
+ * </vaadin-split-layout>
+ * ```
+ *
+ * ### Resize Notification
+ *
+ * This element implements `IronResizableBehavior` to notify the nested resizables
+ * when the splitter is dragged. In order to define a resizable and receive that
+ * notification in a nested element, include `IronResizableBehavior` and listen
+ * for the `iron-resize` event.
```

Comment:
This should probably be removed (or at least rephrased)

Reply:
Removed. I don't think we want to highlight `IronResizableBehavior`.